UnitTests/GitUI.Tests/CommandsDialogs/SettingsDialog/Pages/ColorsPageSettingsPageControllerTests.cs

```diff
@@ -34,17 +34,26 @@ public void Setup()
         }
 
         [Test]
-        public void When_current_theme_is_default_choosing_visual_style_or_theme_variation_should_be_disabled()
+        public void When_current_theme_is_default_choosing_visual_style_should_be_disabled()
         {
             AppSettings.ThemeId = ThemeId.Default;
             ThemeModule.TestAccessor.ReloadThemeSettings(_context.ThemeRepository);
 
             _context.Controller.ShowThemeSettings();
-
-            _context.Page.IsChoosingThemeVariationsEnabled.Should().BeFalse();
             _context.Page.IsChoosingVisualStyleEnabled.Should().BeFalse();
         }
 
+        [Test]
+        public void When_current_theme_is_default_choosing_theme_variation_should_be_enabled()
+        {
+            AppSettings.ThemeId = ThemeId.Default;
+            ThemeModule.TestAccessor.ReloadThemeSettings(_context.ThemeRepository);
+
+            _context.Controller.ShowThemeSettings();
+
+            _context.Page.IsChoosingThemeVariationsEnabled.Should().BeTrue();
```

Comment:
IsChoosingThemeVariationsEnabled is always enabled, property can be removed

Reply:
done